ui/src/variables/utils/buildVarsOption.ts

```diff
@@ -20,7 +20,7 @@ export const buildVarsOption = (variables: VariableAssignment[]): File => ({
         },
         init: {
           type: 'ObjectExpression',
-          properties: variables.map(assignmentToProperty),
+          properties: variables.filter(v => !!v).map(assignmentToProperty),
```

Comment:
and this makes sure those variables we removed are filtered from the list before further processing